### PR TITLE
Circos training papercuts

### PR DESCRIPTION
**Pull request by gallardoalba**
This pull request fixes some problems in the workflow (#1714) and updates the tutorial in order to adapt it to the latest tool version.

### CODE REVIEW

**Comment by hexylena** [comment]
I'll review this tomorrow, super exciting. thanks for contributing these fixes!

File: topics/visualisation/tutorials/circos/tutorial.md [hunk 1]
```diff
@@ -273,29 +274,39 @@ In this section we will reproduce this Circos plot step by step.
...
+> 1. {% tool [Concatenate datasets tail-to-head](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_cat/0.1.0) %}:
+>     - {% icon param-file %} *"Concatenate Datasets"*: `hg18_karyotype.txt`
+>     - {% icon param-repeat %} *"Insert Dataset"*
+>         - In *"1. Dataset"*:
+>             - {% icon param-file %} *"Select"*: `hg18_karyotype_bands.tsv`
+>
+> 2. **Rename** {% icon galaxy-pencil%} the output as `hg18_karyotype_combined.txt`
+>
+>    {% include snippets/rename_dataset.md %}
+{: .hands_on}
```
Comment (hexylena) [inline topics/visualisation/tutorials/circos/tutorial.md:288]: Ahh we had previously intentionally separated it. I think it makes sense to update the zenodo record here and just concatenate the two. @shiltemann that record is yours, could you concatenate the karyotype with the bands file? (just add to the karyotype file is probably best)

File: topics/visualisation/tutorials/circos/tutorial.md [hunk 2]
```diff
@@ -273,29 +274,39 @@ In this section we will reproduce this Circos plot step by step.
...
 >        - In *"Cytogenic Bands"*:
->            - {% icon param-file %} *"Cytogenic Bands"*: `hg18_karyotype_bands.tsv`
->            - *"Convert bands from BED format to circos karyotype band format"*: `No`
->            - *"Fill Bands"*: `2`
+>            - *"Bands transparency"*: `2`
 >            - *"Band Stroke Thickness"*: `1`
 >
 > 2. **Rename** {% icon galaxy-pencil%} the output `Circos Plot ideogram`
 >
->    {% include snippets/rename_dataset.md %}
```
Comment (hexylena) [inline topics/visualisation/tutorials/circos/tutorial.md:298]: I think I'd leave this in (and add name="Circos Plot ideogram")

**Review by hexylena — APPROVED** [review]
I've made a couple changes  but overall it looks good! Thank you for building workflows and updating the tutorial to match the newer tool.

File: topics/visualisation/tutorials/circos/tutorial.md (hunk 1 above)
Comment (gallardoalba) [reply topics/visualisation/tutorials/circos/tutorial.md:288]: I tried to use only the karyotype, but it didn't work for me.

File: topics/visualisation/tutorials/circos/tutorial.md (hunk 2 above)
Comment (gallardoalba) [reply topics/visualisation/tutorials/circos/tutorial.md:298]: Sure :)

File: topics/visualisation/tutorials/circos/tutorial.md (hunk 1 above)
Comment (hexylena) [reply topics/visualisation/tutorials/circos/tutorial.md:288]: Yeah, I remember. My suggestion here to @shiltemann is that we take the karyotype + bands, and produce a single file (now just called hg18_karyotype.txt) so users don't have to merge as the first step (most of the karyotypes available from within circos are like this anyway)

**Review by hexylena — APPROVED** [review]

**Comment by hexylena** [comment]
Thank you so much @gallardoalba!

**Pull request merged**